Java/main-sample/src/main/java/com/perfectomobile/appTest/MyApp.java

```diff
@@ -175,7 +178,7 @@ public static void main(String[] args) throws IOException {
                 driver.close();
 
                 // In case you want to download the report or the report attachments, do it here.
-                PerfectoLabUtils.downloadReport(driver, "pdf", "C:\\test\\report");
+                //PerfectoLabUtils.downloadReport(driver, "pdf", "C:\\test\\report");
```

Comment:
isnt download report uses DigitalZoom pdf?

Reply:
downloadReport in PerfectoLabUtils has it's own method

Reply:
i think this should stay. Yuri implemented it to download pdf from reportium

Reply:
this is old code, yuri confrimed it's not reallu working for cloud that have reportium  enabeld